toolkit/src/styles/components/list-item.scss

```diff
@@ -62,3 +62,39 @@ button.cut-list-item__content-container {
     }
   }
 }
+
+.cut-list-item-template {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 16px;
+}
+
+.cut-list-item-label {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  .title-row {
+    display: flex;
+    flex-direction: row;
+    gap: 8px;
+    align-items: center;
+  }
+}
+
+.cut-list-item-label--title {
```

Comment:
this is not a variant, but an element (from what I understand) so it should maybe be `.cut-list-item-label__title`

(check also other class names, they have a similar issue)

references:
- https://css-tricks.com/bem-101/
- https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/